GuidedArterySegmentation/GuidedArterySegmentation.py

```diff
@@ -835,10 +835,10 @@ def findWidgets(self):
     findWidgets() must have been called first.
     Must be called when the first used effect is activated.
     """
-    def resetMaskingWidgets(self):
+    def setMaskingOptions(self):
```

Comment:
By default, the 'Modify other segments' option is 'OverwriteAllSegments'. Here it is set to 'OverwriteNone', that's why I preferred this name. Am I mistaken about the default values for this option ?

Reply:
The name is not good because it does not tell what the method does. It sets the masking options... to what? You can use `setMaskingOptionsToAllowOverlap` or something like that.

Reply:
OK, I understand, I'll fix.